lib/github/api.rb

```diff
@@ -127,14 +136,37 @@ def get_next_page(header)
     end
 
     def save_repos_to_disk(repos)
-      cache = LocalStorage.new(ConfigPath.new(CACHE_FILE_NAME).get)
-      cache.put(repos.to_json)
+      cache_path = ConfigPath.new(CACHE_FILE_NAME).get
+      cache = LocalStorage.new(cache_path, serialize: false)
```

Comment:
What do you think of moving these local variables to be instance variables? So they don't get initialized on every call to `save_repos_to_disk()` and in other places where they could be reused. They could be lazy virtual attributes like:

```ruby
def cache_path
  @cache_path ||= ConfigPath.new(CACHE_FILE_NAME)
end

def cache_storage
  @cache_storage ||= LocalStorage.new(cache_path, serialize: false)
end
```

It's not a big deal if we leave them here also.